lib/widgets/organization_list.dart

```diff
@@ -32,17 +34,10 @@ class OrganizationList extends StatelessWidget {
           Future<QueryResult?> Function()? refetch,
         }) {
           if (result.hasException) {
-            final isException = databaseFunctions.encounteredExceptionOrError(
+            databaseFunctions.encounteredExceptionOrError(
               result.exception!,
-              showSnackBar: false,
+              showSnackBar: true,
             );
-            if (isException != null) {
-              if (isException) {
-                refetch!();
-              } else {
-                refetch!();
-              }
-            }
```

Comment:
Why are these lines removed?

Reply:
They would have prompted the snackbar again and again.

Reply:
Ok! got it. Go ahead.